charts/k8s-service/README.md

```diff
@@ -48,6 +49,13 @@ The following resources will be deployed with this Helm Chart, depending on whic
 - `ManagedCertificate`: The `ManagedCertificate` is a [GCP](https://cloud.google.com/) -specific resource that creates a Google Managed SSL certificate. Google-managed SSL certificates are provisioned, renewed, and managed for your domain names. Read more about Google-managed SSL certificates [here](https://cloud.google.com/load-balancing/docs/ssl-certificates#managed-certs). Created only if you configure the `google.managedCertificate` input (and set
                          `google.managedCertificate.enabled = true` and `google.managedCertificate.domainName = your.domain.name`).
 
+back to [root README](/README.adoc)
```

Comment:
This is a GREAT idea. We should do this in all our repos.

Reply:
We can do it incrementally. AFAIK I started doing it in aws-service-catalog/tls-scripts, and here. So it's just two places for now.